README.md

```diff
@@ -92,6 +92,16 @@ func demoFunc() {
 	fmt.Println("Hello World!")
 }
 
+type run struct {
+    i int32
+	wg  *sync.WaitGroup
```

Comment:
it needs to gofmt here.

Reply:
@hawkingrei 
Yes, found this formatting issue.
Since it's a markdown file, I fixed it manually